test/CAPI/CMakeLists.txt

```diff
@@ -11,4 +11,6 @@ target_link_libraries(circt-capi-ir-test
 
   MLIRCAPIRegistration
   MLIRPublicAPI
+  CIRCTCAPISV
+  CIRCTCAPIExportVerilog
```

Comment:
Since these libraries are trivial, I don't think it makes sense to create specific test cases for them, though it is valuable to make sure they build.

Reply:
I agree